packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -218,4 +207,36 @@ class ShorebirdFlutter {
     shorebirdEnv.flutterRevision = revision;
     useFlutterProgress.complete();
   }
+
+  /// Helper method to return the human readable version from the
+  /// current flutter revision in [ShorebirdEnv.flutterRevision].
+  Future<String> humanReadableVersionFromEnv() async {
+    return humanReadableVersion(
+      flutterRevision: shorebirdEnv.flutterRevision,
+    );
+  }
```

Comment:
What's the value of having an extra level of indirection here? I'd delete this helper method and just pass `shorebirdEnv.flutterRevision` when needed.